beacon-chain/rpc/endpoints_test.go

```diff
@@ -31,6 +31,7 @@ func Test_endpoints(t *testing.T) {
 		"/eth/v2/beacon/blinded_blocks":                              {http.MethodPost},
 		"/eth/v1/beacon/blocks":                                      {http.MethodPost},
 		"/eth/v2/beacon/blocks":                                      {http.MethodPost},
+		"/eth/v2/beacon/blobs":                                       {http.MethodPost},
```

Comment:
path would be affected based on my previous comment

Reply:
+1

Reply:
Updated